plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManagerTest.kt

```diff
@@ -319,6 +319,8 @@ class LocationPuckManagerTest {
   @Test
   fun testDefault3DStyleScalingWithMercatorScale() {
     val onLocationUpdatedSlot = slot<((Point) -> Unit)>()
+    val getStyleSlot = slot<((StyleInterface) -> Unit)>()
+    val style = mockk<StyleInterface>(relaxed = true)
```

Comment:
nit: remove relaxed = true and mock all the function calls.

Reply:
discussed offline that test should be slightly revisited in general